### ISSUE TITLE
Maximum execution time exceeds

### ISSUE THREAD

**Issue by mverma16** [ladybirdweb/faveo-helpdesk#454]
- Faveo Version : all version
- PHP version : 5.6+
- Database Driver & Version : MySql
- Server specification :

#### Description:
In installation step5, installer runs migrations and seeds the database with default configuration data. The required max_execution time specified by the installer is 120 seconds, on some servers even this time limits exceeds while installation and the system throws maximum execution time exceed error.

Ideally every script should not take more than 30 seconds but considering 120 seconds as except able max_execution time, system should not throw any error regarding max_execution time.

In other words all script needs to be optimized so that they don't exceed maximum execution time and can be run without any problem on any server. 

#### Downloaded from
- [x] master-branch

- [x] development-branch

- [x] release-tag
